pennylane/templates/layers/random.py

```diff
@@ -99,10 +94,116 @@ def RandomLayers(weights, wires, ratio_imprim=0.3, imprimitive=CNOT, rotations=N
+
+    .. UsageDetails::
+
+        **Default seed**
+
+        ``RandomLayers`` always uses a seed to initialize the construction of a random circuit. If no seed is
+        provided, the default seed of ``42`` is used.
+
+        This means that the template creates the same circuit every time it is called without specifying a seed.
+
+        .. code-block:: python
+
+            import pennylane as qml
+            import numpy as np
+            from pennylane.templates.layers import RandomLayers
+
+            dev = qml.device("default.qubit", wires=2)
+            weights = [[0.1, -2.1, 1.4]]
+
+            @qml.qnode(dev)
+            def circuit1(weights):
+                RandomLayers(weights=weights, wires=range(2))
+                return qml.expval(qml.PauliZ(0))
+
+            @qml.qnode(dev)
+            def circuit2(weights):
+                RandomLayers(weights=weights, wires=range(2))
+                return qml.expval(qml.PauliZ(0))
+
+        >>> np.allclose(circuit1(weights), circuit2(weights))
+        >>> True
+
+        You can verify this by drawing the circuits.
+
+            >>> print(circuit1.draw())
+            >>>  0: ──RX(0.1)──RX(-2.1)──╭X──╭X───────────┤ ⟨Z⟩
+            ...  1: ─────────────────────╰C──╰C──RZ(1.4)──┤
+
+            >>> print(circuit2.draw())
+            >>>  0: ──RX(0.1)──RX(-2.1)──╭X──╭X───────────┤ ⟨Z⟩
+            ...  1: ─────────────────────╰C──╰C──RZ(1.4)──┤
+
+        **Changing the seed**
+
+        To change the randomly generated circuit architecture, you have to change the seed passed to the template.
+        For example, these two calls of ``RandomLayers`` *do not* create the same circuit:
+
+        .. code-block:: python
+
+            @qml.qnode(dev)
+            def circuit_9(weights):
+                RandomLayers(weights=weights, wires=range(2), seed=9)
+                return qml.expval(qml.PauliZ(0))
+
+            @qml.qnode(dev)
+            def circuit_12(weights):
+                RandomLayers(weights=weights, wires=range(2), seed=12)
+                return qml.expval(qml.PauliZ(0))
+
+        >>> np.allclose(circuit_9(weights), circuit_12(weights))
+        >>> False
+
+        >>> print(circuit_9.draw())
+        >>>  0: ──╭X──RY(-2.1)──RX(1.4)──┤ ⟨Z⟩
+        ...  1: ──╰C──RX(0.1)────────────┤
+
+        >>> print(circuit_12.draw())
+        >>>  0: ──╭X──RX(-2.1)──╭C──╭X──RZ(1.4)──┤ ⟨Z⟩
+        ...  1: ──╰C──RZ(0.1)───╰X──╰C───────────┤
+
+
+        **Automatically creating random circuits**
+
+        To automate the process of creating different circuits with ``RandomLayers``,
+        you can pass a random number generator as seed. However, in this case care needs
+        to be taken. In the default setting, a quantum node is **mutable**, which means that the quantum function is
+        re-evaluated every time it is called. This means that the circuit is re-constructed from scratch
+        each time you call the qnode:
```

Comment:
Instead of passing a random number generator, could the user just do `seed=None`?

Reply:
I'm so blind, of course, that should work! Will update.